### PR TITLE
Update changes.md

### LINKED ISSUE #2521: Check Changelog against all recent PR's
I've been bad and failed to put information about my recent flurry of changes into the changelog.

These changes for Rose 2.0rc1 should be added. ([List of closed rc1 issues](https://github.com/metomi/rose/pulls?q=is%3Apr+milestone%3A2.0rc1+is%3Aclosed))

Probably not worthwhile until [the list of functional tickets](https://github.com/metomi/rose/issues?q=is%3Aopen+is%3Aissue+milestone%3A2.0rc1+-label%3Adoc+) is cleared.

### PR DESCRIPTION
**Pull request by wxtim**
I missed updating the changelog for a bunch of PR's. This one should fix.

Closes #2521

Waiting on 
- [x] [#2510]
- [x] [#2514]
- [ ] [#2531]
- [x] [#2519]
- [x] [#2522]
- [x] [#2528]

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -8,6 +8,34 @@ creating a new release entry be sure to copy & paste the span tag with the
 `actions:bind` attribute, which is used by a regex to find the text to be
 updated. Only the first match gets replaced, so it's fine to leave the old
 ones in. -->
+--------------------------------------------------------------------------------
+
+## 2.0b2 (<span actions:bind='release-date'></span>)
```
[the remark this answers is not in the document]
Comment (wxtim) [reply CHANGES.md:13]:
```suggestion
## 2.0rc1 (<span actions:bind='release-date'></span>)
```

[the remark this answers is not in the document]
Comment (wxtim) [reply CHANGES.md]: Correctly copied and pasted. Not correctly modified afterwards.

File: CHANGES.md [hunk 2]
```diff
@@ -8,6 +8,34 @@ creating a new release entry be sure to copy & paste the span tag with the
...
+[#2522](https://github.com/metomi/rose/pull/2522) -
+Removed the Rsync implied by the `prune-remote-logs-at` option of
+Rose prune.
+
+[#2528](https://github.com/metomi/rose/pull/2528) -
+Rose Date only emits a warning if running in interactive mode.
+
+
+
+
```
Comment (oliver-sanders) [inline CHANGES.md:38]:
```suggestion
```

**Review by oliver-sanders — APPROVED** [review]

File: CHANGES.md [hunk 3]
```diff
@@ -8,6 +8,32 @@ creating a new release entry be sure to copy & paste the span tag with the
...
+[#2519](https://github.com/metomi/rose/pull/2519) -
+Updated the Rose documentation to describe using
+`cylc validate; cylc install; cylc play` instead of `rose suite-run`.
+
+[#2522](https://github.com/metomi/rose/pull/2522) -
+Removed the Rsync implied by the `prune-remote-logs-at` option of
+Rose prune.
+
+[#2528](https://github.com/metomi/rose/pull/2528) -
+Rose Date only emits a warning if running in interactive mode.
```
Comment (oliver-sanders) [inline CHANGES.md:34]: Easier to do this in the relevant PR?

File: CHANGES.md [hunk 4]
```diff
@@ -8,6 +8,32 @@ creating a new release entry be sure to copy & paste the span tag with the
...
+[#2531](https://github.com/metomi/rose/pull/2531) -
+Rosie Graph disabled.
+
+[#2519](https://github.com/metomi/rose/pull/2519) -
+Updated the Rose documentation to describe using
+`cylc validate; cylc install; cylc play` instead of `rose suite-run`.
+
+[#2522](https://github.com/metomi/rose/pull/2522) -
+Removed the Rsync implied by the `prune-remote-logs-at` option of
+Rose prune.
```
Comment (oliver-sanders) [inline CHANGES.md:31]: Here too.

**Review by oliver-sanders — CHANGES_REQUESTED** [review]
Whoops, sorry, not ready for merging.

File: CHANGES.md (hunk 3 above)
Comment (wxtim) [reply CHANGES.md:34]: This has now been merged, so no.

File: CHANGES.md (hunk 4 above)
Comment (wxtim) [reply CHANGES.md:31]: At this stage I think it will avoid conflicts.

**Review by oliver-sanders — APPROVED** [review]

**Pull request merged**